tools/doxygen/doxygen.cmake

```diff
@@ -480,6 +480,10 @@ function(make_doxygen_target modules_var)
         WORKING_DIRECTORY ${output_dir}
         COMMENT "Generating Inviwo API documentation with Doxygen"
         VERBATIM
+        COMMAND ${CMAKE_COMMAND} -E copy "${IVW_ROOT_DIR}/docs/css/doxygen.css" "${output_dir}/inviwo/html/doxygen.css"
+        COMMAND ${CMAKE_COMMAND} -E copy "${IVW_ROOT_DIR}/docs/css/navtree.css" "${output_dir}/inviwo/html/navtree.css"
+        COMMAND ${CMAKE_COMMAND} -E copy "${IVW_ROOT_DIR}/docs/css/search.css" "${output_dir}/inviwo/html/search.css"
+        COMMAND ${CMAKE_COMMAND} -E copy "${IVW_ROOT_DIR}/docs/css/tabs.css" "${output_dir}/inviwo/html/tabs.css"
```

Comment:
I think this can be specified in the generated doxy configuration file using HTML_STYLESHEET and HTML_EXTRA_STYLESHEET (http://www.doxygen.nl/manual/config.html#cfg_html_stylesheet) http://www.doxygen.nl/manual/config.html#cfg_html_extra_stylesheet

Reply:
yes

Reply:
yes, we found this and prepared an extra stylesheet, however doxygen did not seem to apply the stylesheet properly and there were always things missing, that is why we did the copying in the end

Reply:
Make a note explaining that...